ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2399,6 +2399,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-compact
+- as: vsphere-8-ipi-p2-f7
```

Comment:
I think we can reuse `vsphere-ipi-ovn-etcd-encryption-p2-f7` by switching the cluster profile to `vsphere-8`.